org.overte.Overte.yml

```
@@ -0,0 +1,70 @@
+app-id: org.overte.Overte
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+# QtWebEngine is not part of the KDE runtime,
+# so we use this package, which is also maintained by KDE developers.
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.15-23.08'
+command: Overte
+separate-locales: false
+finish-args:
+  - --share=network
+  - --share=ipc # For Discord integration. Also required by X11.
+  - --device=dri # Graphics driver
+  - --socket=pulseaudio
+  # Overte requires OpenGL ES 3.2. While most Mesa drivers don't have this fully implemented,
+  # we try anyway, since it works on some devices.
+  - --env=MESA_GLES_VERSION_OVERRIDE=3.2
+  # QtWebEngine is shipped by the baseapp and not by the Sdk, which makes it use a non-standard path.
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  # KDE seems to do some aggressive theming which breaks things.
+  # See issue: https://github.com/overte-org/overte/issues/167
+  - --env=XDG_CURRENT_DESKTOP=foo
+  # While we technically support Wayland, there are issues with mouse capture: https://github.com/overte-org/overte/issues/628
+  - --socket=x11
+  - --env=QT_QPA_PLATFORM=xcb
+modules:
+  - name: Overte
+    buildsystem: simple
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://public.overte.org/build/overte/release/2024.07.1/Overte-2024.07.1-x86_64.AppImage
+        sha256: f916ea2b9c6d6cd7667cb8cf2cbf3fde9fbe261031bc17072b93ab45cb4c9e2b
+        dest-filename: Overte.AppImage
+        # This update checker may be unreliable.
+        x-checker-data:
+          type: html
+          url: https://overte.org/downloads.html
+          version-pattern: "The current release of Overte is version ([\\d\\.-]+)"
+          url-pattern: "(https://public.overte.org/build/overte/release/([\\d\\.-]+)/Overte-([\\d\\.-]+)-x86_64.AppImage)"
+
+      - type: file
+        only-arches:
+          - aarch64
+        url: https://public.overte.org/experiment/overte/2024.07.1/Overte-2024.07.1-modified-colorspace-aarch64.AppImage
+        sha256: 391b45f36ac8d4441ca0b532abc48d5c1ce5c9ff067c6d1901338ba099640912
+        dest-filename: Overte.AppImage
+
+      - type: script
+        dest-filename: Overte
+        commands:
+          - exec /app/bin/overte/interface/interface "$@"
+
+      - type: file
+        path: org.overte.Overte.appdata.xml
+
+    build-commands:
+      - chmod +x Overte.AppImage
+      - ./Overte.AppImage --appimage-extract
+      - mv squashfs-root/* ${FLATPAK_DEST}/bin/
```

I am pretty sure this should be 
```suggestion
      - mv squashfs-root/* ${FLATPAK_DEST}/
```

---

It's deliberate. The AppImage isn't exactly very clean on the inside, as it is ment to be self-contained. Even putting it outside of bin/, it would not integrate with the Flatpaks directory structure, so it doesn't really matter where it is put.